examples/src/main/java/org/apache/spark/examples/ml/JavaGaussianMixtureExample.java

```diff
@@ -54,6 +54,7 @@ public static void main(String[] args) {
 
     // Output the parameters of the mixture model
     for (int i = 0; i < model.getK(); i++) {
+      System.out.println("Gaussian " + i);
```

Comment:
Yeah the 2 print statements could be combined.  I was probably just trying not to cram too much together, but I think it would be fine.